### PR TITLE
My first pull request

### PR DESCRIPTION
**Pull request by Sonickmumba**
I have fixed the Linter errors and also added a session for getting started in the README.md file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,73 +1,27 @@
...
-
-### Setup
-
-### Install
-
-### Usage
-
-### Run tests
-
-### Deployment
```
Comment (codecaiine) [inline README.md]: - [ ] You README file is descriptive, you did well, but please add another session for getting started to show how use your project.

**Review by codecaiine — CHANGES_REQUESTED** [review]

**Comment by codecaiine** [comment]
Hi @Sonickmumba , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
:heavy_check_mark:  Good commit history
:heavy_check_mark: Descriptive commits
:heavy_check_mark: Good project implementation

## Required Changes ♻️

- [ ]  Please a summary describing introduced changes in your pull request with more details.
- [ ] Please Fix your Linters errors cause the result should be green. 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Sonickmumba** [comment]
Dear @codecaiine,
I have managed to fix the linter errors and added the session for getting started on the README.md file. Kindly check it out.

Thanks and regards.

**Review by sumairq — APPROVED** [review]
### Hi @Sonickmumba     , 



Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ Good commit history
✔️Linters are passing
✔️Very professional README
✔️Correct Github flow
✔️Descriptive PR

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**